src/src/editor/D2CompilerState.kt

```diff
@@ -13,6 +13,7 @@ internal class D2CompilerState(
   val port: Int,
   val theme: String?,
   val layout: D2Layout?,
+  val sketch: Boolean?,
```

Comment:
Can we switch to a nonnull type `Boolean` here?

Reply:
I'll give it a try. This is my first time with Kotlin 😄

Reply:
Achieved 😄